pkg/suggestion/v1alpha3/chocolate/base_chocolate_service.py

```diff
@@ -1,11 +1,8 @@
-import json
-import numpy as np
 import chocolate as choco
 import logging
 import base64
 
-from pkg.suggestion.v1alpha3.internal.search_space import *
-from pkg.suggestion.v1alpha3.internal.trial import *
+from pkg.suggestion.v1alpha3.internal import *
```

Comment:
In some files we are using directly functions/variables import like here: https://github.com/kubeflow/katib/blob/master/pkg/suggestion/v1alpha3/chocolate_service.py#L8, but here we are using `import *`.
Maybe in all python files we should avoid `import *` and use only relevant imports, so we can easily see which method we are using from `search_space` and `trial` file?

What do you think?

Reply:
> Maybe in all python files we should avoid import * and use only relevant imports, so we can easily see which method we are using from search_space and trial file?

Yes, I agree you opinion. star-imports are known as anti-pattrens in Python.
OK, I'll remove star-imports.